### PR TITLE
Add Ishu Singhal Pune file

### PR DESCRIPTION
**Pull request by ishusinghal**
- [x] Add commit
- [x] Review comment
- [ ] official review
- [ ] Merge

### CODE REVIEW

**Comment by vedashreevinay** [comment]
Just a general comment!

File: ishusinghal-pune.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Pune, India
```
Comment (vedashreevinay) [inline ishusinghal-pune.md:1]: Should be a double hash!

File: ishusinghal-pune.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+### Ishu Singhal 1
+
+
+## Ishu Singha123
+
+# Ishu 123
```
Comment (ishusinghal) [inline ishusinghal-pune.md]: :rofl:  :+1:

File: ishusinghal-pune.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# Pune, India
+
+###Places avoid to Visit
+East country because its too hot than the pune more than 20 degree
```
Comment (vedashreevinay) [inline ishusinghal-pune.md:4]: Great!

**Review by vedashreevinay — APPROVED** [review]

**Review by ishusinghal — COMMENTED** [review]
Looks Great

**Pull request merged**